checkstyle/src/main/resources/vitro-checkstyle/checkstyle-suppressions.xml

```diff
@@ -3,6 +3,8 @@
     "-//Puppy Crawl//DTD Suppressions 1.1//EN"
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 <suppressions>
+  <suppress files=".+\.(?:txt|xml|csv|sh|thrift|html|sql|eot|ttf|woff|css|png|java|jar|zip|war|class|tar|bin|gif|jpg|jpeg|properties)$" checks=".*"/>	
```

Comment:
I would suggest removing this modification.

Reply:
I stand corrected. There are four more checkstyle errors in another update in the VIVO pull-request associated with this [current ticket](https://github.com/vivo-project/VIVO/pull/136/files#diff-1484b49dd85aae657f181084bc7c2f93R377-R382).
```
[ERROR] src/main/resources/config/example.runtime.properties:[377] (regexp) RegexpSingleline: Line has trailing whitespace
[ERROR] src/main/resources/config/example.runtime.properties:[378] (regexp) RegexpSingleline: Line has trailing whitespace
[ERROR] src/main/resources/config/example.runtime.properties:[379] (regexp) RegexpSingleline: Line has trailing whitespace
[ERROR] src/main/resources/config/example.runtime.properties:[380] (regexp) RegexpSingleline: Line has trailing whitespace
```